lint/util.py

```diff
@@ -216,14 +218,27 @@ def communicate(cmd, code=None, output_stream=STREAM_STDOUT, env=None, cwd=None)
     else:
         stdout = stderr = None
 
-    out = popen(cmd, stdout=stdout, stderr=stderr,
-                output_stream=output_stream, env=env, cwd=cwd)
+    proc = popen(cmd, stdout=stdout, stderr=stderr,
+                 output_stream=output_stream, env=env, cwd=cwd)
+
+    if proc is not None:
+        from . import persist
```

Comment:
I got weird feeling it should be on top of the file...

Reply:
I just wrote this down very quickly, and surprisingly it worked on Windows, so I thought I push it to get some feedback.

Reply:
It doesn't matter where imports are located in most cases.
The convention is to put them at the top so they can be used throughout the entire module, but sometimes late imports might be necessary to prevent circular imports.

I didn't check whether that is the case here.